doc/python/bar-charts.md

```diff
@@ -304,6 +304,57 @@ fig.update_layout(barmode='stack')
 fig.show()
 ```
 
+### Stacked Bar Chart from aggregating a data frame 
+
+Stacked bar charts are a powerful way to present results summarizing categories generated using the Pandas aggregate commands.  DF.agg() which produces a wide format data set with one row for each bar component and a column for each bar, which is the transpose of the orientation of the px.bar wide data frame.  Tranposing and updating the indexes is a somewhat involved option. Here is one straightforward way to aggregate a data set into a summarized form and present the results as a stacked bar.
+
```

Comment:
```suggestion
Stacked bar charts are a powerful way to present results summarizing categories generated using the Pandas aggregate commands. `pandas.DataFrame.agg` produces a wide format data set with one row for each bar component and a column for each bar, which is the transpose of the orientation of the `px.bar` wide data frame.  Transposing and updating the indexes is a somewhat involved option. Here is one straightforward way to aggregate a data set into a summarized form and present the results as a stacked bar using `plotly.graph_objects`.

```

Thanks for the PR @rl-utility-man! Great example. 
I edited to provide some full names of the functions. 

Do you think changing the following 

```
`pandas.DataFrame.agg` produces a wide format data set with one row for each bar component and a column for each bar, which is the transpose of the orientation of the `px.bar` wide data frame.  
```
to
```
`pandas.DataFrame.agg` produces a wide format data set that doesn't work directly with `px.bar`.
```
retains the meaning you want to convey?

Reply:
Glad you liked this @LiamConnors!  Yes, shortening that detailed sentence is an improvement. Consider getting the reversed/transposed idea in succinctly.

How about:

`pandas.DataFrame.agg` produces a wide format data set with the rows and columns reversed from the `px.bar` expectation.

Reply:
Thanks @rl-utility-man. That sounds good to me. I've updated my commit suggestion based on it. What do you think?

Reply:
Looks great @LiamConnors.  My only remaining suggestion is to rewrite the last sentence to:

 Here is one straightforward alternative, which presents the aggregated data as a stacked bar using `plotly.graph_objects`.